src/query/service/src/api/rpc/packets/packet_data.rs

```diff
@@ -172,7 +183,8 @@ impl TryFrom<FlightData> for DataPacket {
                     progress: progress_info,
                 })
             }
-            0x05 => Ok(DataPacket::ClosingClient),
+            0x05 => Ok(DataPacket::ClosingOutput),
```

Comment:
Do we need to add enum for them?

Reply:
Enum nesting without any field may be meaningless.